src/resources/js/v2/tickets-block.js

```diff
@@ -637,7 +643,7 @@ tribe.tickets.block = {
 		const $parent = $trigger.closest( obj.selectors.item );
 		const $target = $( '#' + $trigger.attr( 'aria-controls' ) );
 
-		if ( ! $target || ! $parent ) {
+		if ( ! $target.length || ! $parent ) {
```

Comment:
shouldn't both of these have the `length` property checked?

Reply:
@paulmskim opened a PR -> https://github.com/moderntribe/event-tickets/pull/1848